### DISCUSSION TITLE
Renovate bumps to unstable version

### DISCUSSION THREAD

**Discussion by simoneb** [renovatebot/renovate#8018] [Suggest an Idea]
[Here's my Renovate config](https://github.com/nearform/titus/blob/master/renovate.json), which has not changed for the past month or so.

I understand that the configuration should not bump to versions which are not the latest (respectLatest is true by default and I don't see anything in the config setting it to false), plus the base config sets ignoreUnstable to true, too.

Yet, renovate bumped dependency Husky to an unstable version having the `next` tag in this PR https://github.com/nearform/titus/pull/640. 

Husky's latest is 4.x, next is 5.x. 

Why did Renovate upgrade to the next tag?

**Comment by rarkins** [comment]
Because 5.x was temporarily tagged as `latest`. See https://github.com/typicode/husky/issues/825

**Comment by rarkins** [comment]
It looks like you have automerge enabled even for that major update - I recommend you don't do that.

**Comment by simoneb** [comment]
@rarkins thanks for the explanation, I missed that! And thanks for the advice, our decision was intentional as we want to limit the maintenance required on that project. I mitigated the possibility of similar issues in the future by introducing stabilityDays in the config. Thanks again for your support.
